extensions/zig/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zed_zig"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Can we leave the version bump out of this PR?

We bump the versions separately.

Reply:
Taken care of